cpp/mrc/include/mrc/manifold/composite_manifold.hpp

```diff
@@ -65,7 +73,7 @@ class CompositeManifold : public Manifold
     }
 
   private:
-    void do_add_input(const SegmentAddress& address, node::SourcePropertiesBase* input_source) final
+    void do_add_input(const SegmentAddress& address, node::IIngressAcceptorBase* input_source) final
```

Comment:
Base or Handle for the fully type-erased class.  In channel we have IngressHandle.

probably time for consistency.

I thin Base for classes that provide function and Handle for those that are used an opaque "handles".

Reply:
Went with Base since that was the naming scheme from before.